.php_cs

```diff
@@ -21,40 +21,80 @@ $config->setRules(
         'array_indentation' => true,
         'array_syntax' => ['syntax' => 'short'],
         'binary_operator_spaces' => true,
+        'blank_line_after_opening_tag' => true,
         'blank_line_before_statement' => [
-            'statements' => ['return', 'continue', 'case', 'default', 'declare', 'throw', 'try'],
+            'statements' => [
+                'break',
+                'case',
+                'continue',
+                'declare',
+                'default',
+                'die',
+                'do',
+                'exit',
+                'for',
+                'foreach',
+                'goto',
+                'if',
+                'include',
+                'include_once',
+                'require',
+                'require_once',
+                'return',
+                'switch',
+                'throw',
+                'try',
+                'while',
+                'yield',
```

Comment:
Disagree with:

- `break` (we already put line before `case`)
- `include`
- `include_once`
- `require`
- `require_once`

But the rest 🙌

Reply:
re: `(includ|requir)e(_once)?` ... I kinda don't mind - just that we have one around the block of them...

Reply:
It would be preferable if a single line could be applied before a series of these and not each individual one but I just don't think it's worth it. Those statements are only used in some bootstrap files and configs that are rarely touched.